src/components/implementation/no_interface/scheddev/fprr/sl_timer_mod_periodic.c

```diff
@@ -0,0 +1,78 @@
+#include <sl.h>
+#include <sl_consts.h>
+#include <sl_plugins.h>
+#include <heap.h>
+
+#define SL_TIMEOUT_MOD_MAX_THDS MAX_NUM_THREADS
+
+struct wakeup_heap {
+	struct heap  h;
+	void        *data[SL_TIMEOUT_MOD_MAX_THDS];
+	char         p; /* pad. TODO: use alignment */
```

Comment:
This was from the example I picked up from another usage of heap library, I didn't change it. I added a TODO for using alignments. I'm stupid, trying to get changes in quick! 

This padding is not required at all here, I will remove it.